src/transport/multicast/read.c

```diff
@@ -117,5 +137,26 @@ void *_zp_multicast_read_task(void *ztm_arg) {
 
     return NULL;
 }
+#else
+int8_t _zp_multicast_read(_z_transport_multicast_t *ztm) {
+    _ZP_UNUSED(ztm);
+    return _Z_ERR_TRANSPORT_NOT_AVAILABLE;
+}
 
+int _zp_multicast_start_read_task(_z_transport_t *zt, _z_task_attr_t *attr, _z_task_t *task) {
```

Comment:
As convention, return `int8_t` instead.
This goes back to a previous discussion on having a list of return codes.